### DISCUSSION TITLE
Coding standards

### DISCUSSION THREAD

**Discussion by W0lfgunbl00d** [audacity/audacity#7446] [Q&A]
I have read the coding standards specified is the wiki section (https://github.com/audacity/audacity/wiki/Coding-Standards) but it was too specific and i haven't been able to determine wether or not the coding follows an international standard like ISO/IEC. 
I would really appreciate  an answer to this question for my engineering school project.

**Comment by LWinterberg** [comment]
I don't believe there's any standard that specifies 3 spaces as indentation. So no, it doesn't

**Reply by W0lfgunbl00d** [reply]
Thanks
